widelands.svg

```diff
@@ -0,0 +1,41 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
```

Comment:
I suppose it's not part of upstream tarball? Why not ask upstream team to include it? And why bundle PNG if you have SVG at hand?

Reply:
> I suppose it's not part of upstream tarball? Why not ask upstream team to include it? And why bundle PNG if you have SVG at hand?

Tarball contains the icon, but in low resolution (64x64 or 128x128). And if I remember correctly, you require high-resolution PNG (256x256?) and/or SVG.